cmd/runtimetest/main.go

```diff
@@ -11,6 +11,7 @@ import (
 	"syscall"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/mount"
```

Comment:
I think we should probably pull out mount parsing into a single function. Pulling this dependency is adding a lot of code we don't need.